plugins/links.rb

```diff
@@ -26,7 +26,7 @@ def initialize
   end
 
   def on_message(message, matches)
-    url = matches[0]
+    url = matches[0].split('|')[0]
```

Comment:
that regex is awful and needs to be hatcheted, but this will do for now. I'd add a comment explaining where the `|` comes from, since this makes little sense without context.